distributions/standalone/src/main/resources/conf/scheduler-pools.conf

```diff
@@ -2,26 +2,26 @@
 # Certain attributes support expressions. Those are arithmetic expressions (+, -, / or * operators supported only) that are
 # evaluated when the runtime starts.
 # Those expressions may be applied to numbers (integers or decimals) or any of the following variables:
-# * C: the number of processors available to the Java virtual machine.
+# * cores: the number of processors available to the Java virtual machine.
 
-# The maximum time to wait until all tasks in all the runtime thread pools have completed execution when stopping the
-# scheduler service.
-SchedulerService.gracefulShutdownTimeoutSeconds=15
+# The maximum time (in milliseconds) to wait until all tasks in all the runtime thread pools have completed execution when
+# stopping the scheduler service.
+org.mule.runtime.scheduler.gracefulShutdownTimeout=15000
 
 # The number of threads to keep in the cpu_lite pool, even if they are idle.
 # Supports Expressions
-SchedulerService.cpuLight.threadPoolSize=2*C
+org.mule.runtime.scheduler.cpuLight.threadPoolSize=2*cores
 
 # The number of threads to keep in the I/O pool.
 # Supports Expressions
-SchedulerService.io.coreThreadPoolSize=C
+org.mule.runtime.scheduler.io.threadPoolSize.core=cores
```

Comment:
Then we should use:
`threadPool.coreSize`
`threadPool.maxSize`
`threadPool.keepAlive`  (not this should be attribute of theedPool too.)